tests/Integration/SettingsCenterRepositoryTest.php

```
@@ -0,0 +1,272 @@
+        $this->assertWPError($result);
+        $this->assertSame('easymde_settings_invalid_shortcut', $result->get_error_code());
+        $this->assertSame($before, get_option(Options::EDITOR_SETTINGS, array()));
+    }
+
+    public function test_rest_update_requires_manage_options_and_returns_sanitized_settings()
+    {
+        global $wp_rest_server;
+        $wp_rest_server = new WP_REST_Server();
+        do_action('rest_api_init');
+
+        $administrator_id = self::factory()->user->create(array('role' => 'administrator'));
```

PHPStan (2.2.7)

[error] Call to an undefined static method SettingsCenterRepositoryTest::factory().


(staticMethod.notFound)